### ISSUE TITLE
Refuse to open all corrupt objects

### ISSUE THREAD

**Issue by jjatria** [praat/praat#371]
Praat is very strict about the data it handles, and will often refuse to open an object that is corrupt: if a string is expected and a number is found, for example.

However, this is not the case with all types of corruption. In particular, TextGrid files are very sensitive about their timestamps: the end of an interval _must be_ the beginning of the interval that follows. And yet Praat will happily open a TextGrid in which this is not the case.

This leads to bugs that are difficult to track down. It would be better if Praat refused to open these files, maybe with a specifying where the error lies. Otherwise, opening any TextGrid leaves you with the possibility that Praat will behave unexpectedly under correct operation by the user, and there is no way to know this until it does.

**Comment by PaulBoersma** [comment]
Yes, I have agreed with this for a long time. Behaviour of the kind you mention has actually happened with TextGrid file generated by other programs.
